guides/micronaut-security-basicauth/java/src/main/java/example/micronaut/DefaultAuthorizationExceptionHandlerReplacement.java

```diff
@@ -3,23 +3,26 @@
 import io.micronaut.context.annotation.Replaces;
 import io.micronaut.http.HttpRequest;
 import io.micronaut.http.HttpResponse;
-import io.micronaut.http.HttpStatus;
 import io.micronaut.http.MutableHttpResponse;
 import io.micronaut.security.authentication.AuthorizationException;
 import io.micronaut.security.authentication.DefaultAuthorizationExceptionHandler;
-
 import jakarta.inject.Singleton;
 
+import static io.micronaut.http.HttpHeaders.WWW_AUTHENTICATE;
+import static io.micronaut.http.HttpStatus.FORBIDDEN;
+import static io.micronaut.http.HttpStatus.UNAUTHORIZED;
+
 @Singleton // <1>
 @Replaces(DefaultAuthorizationExceptionHandler.class) // <2>
 public class DefaultAuthorizationExceptionHandlerReplacement extends DefaultAuthorizationExceptionHandler {
 
     @Override
-    protected MutableHttpResponse<?> httpResponseWithStatus(HttpRequest request, AuthorizationException exception) {
-        if (exception.isForbidden()) {
-            return HttpResponse.status(HttpStatus.FORBIDDEN);
+    protected MutableHttpResponse<?> httpResponseWithStatus(HttpRequest request,
+                                                            AuthorizationException e) {
+        if (e.isForbidden()) {
```

Comment:
I like the original flow without an else bracket better. I think it is nicer to only remove the return statement when it is the last sentence.

Reply:
did you mean this comment for the Kotlin version? I didn't remove a return statement here

Reply:
Sorry, it seems I miss commented here.